test/DeployCampaignInt.test.ts

```diff
@@ -69,100 +73,84 @@ describe("Deploy Campaign Test", () => {
     [deployAdmin, admin, governor, zeroVault, userA, userB] = await hre.ethers.getSigners();
   });
 
-  describe("MEOW Token Ops", () => {
+  describe("Z Token Ops", () => {
     before(async () => {
-      campaignConfig = {
-        env,
-        deployAdmin,
-        governorAddresses: [deployAdmin.address],
-        adminAddresses: [deployAdmin.address, admin.address],
-        domainToken: {
-          name: ZNS_DOMAIN_TOKEN_NAME,
-          symbol: ZNS_DOMAIN_TOKEN_SYMBOL,
-          defaultRoyaltyReceiver: deployAdmin.address,
-          defaultRoyaltyFraction: DEFAULT_ROYALTY_FRACTION,
-        },
-        rootPriceConfig: DEFAULT_PRICE_CONFIG,
-        zeroVaultAddress: zeroVault.address,
-        stakingTokenAddress: MeowMainnet.address,
-        mockMeowToken: true,
-        postDeploy: {
-          tenderlyProjectSlug: "",
-          monitorContracts: false,
-          verifyContracts: false,
-        },
-      };
+      campaignConfig = await getConfig(
+        {
+          deployer: deployAdmin,
+          governors: [deployAdmin.address],
+          admins: [deployAdmin.address, admin.address],
+          zeroVaultAddress: zeroVault.address,
+          env,
+        }
+      );
+      campaignConfig.domainToken.defaultRoyaltyReceiver = deployAdmin.address;
+      campaignConfig.postDeploy.tenderlyProjectSlug = "";
     });
 
-    it("should deploy new MeowTokenMock when `mockMeowToken` is true", async () => {
+    it("should deploy new ZTokenMock when `mockZToken` is true", async () => {
```

Comment:
this test is not testing anything. it needs to test what you have in the name.
there is no check for the variable value or that the deployed contract is in fact a mock